# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xfce4-terminal/hotfix-disabledoc.patch.cross
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xfce4-terminal-1.1.5/meson.build.vanilla	2025-04-02 23:40:53.050978254 +0200
+++ xfce4-terminal-1.1.5/meson.build	2025-04-02 23:40:55.555445376 +0200
@@ -219,7 +219,6 @@
 endforeach
 
 subdir('colorschemes')
-subdir('doc')
 subdir('icons')
 subdir('po')
 subdir('terminal')
